core/src/main/java/nl/nn/adapterframework/statistics/MetricsInitializer.java

```diff
@@ -102,10 +130,12 @@ public NodeConfig openGroup(NodeConfig parentData, String dimensionName, String
 		String nodeName = parentData.name;
 		List<Tag> tags = new LinkedList<>(parentData.tags);
 		int groupLevel = parentData.groupLevel;
-		if (StringUtils.isNotEmpty(dimensionName)) {
-			tags.add(Tag.of(type, dimensionName));
-		} else {
-			nodeName=nodeName+"."+type;
+		if(type != null) {
```

Comment:
Deze check is nu nodig omdat je in Configuration.initMetrics zo graag null wilde doorgeven, en het in MetricsInitializer met de hand alsnog hebt gedaan.